packages/commercetools/api-client/src/api/createCart/index.ts

```diff
@@ -1,11 +1,9 @@
-import { CustomQueryFn } from '../../index';
-import { getCustomQuery } from '../../helpers/queries';
-import defaultQuery from './defaultMutation';
+import defaultMutation from './defaultMutation';
 import { CartData } from './../../types/Api';
 import gql from 'graphql-tag';
 
-const createCart = async ({ config, client }, cartDraft: CartData = {}, customQueryFn?: CustomQueryFn) => {
-  const { locale, acceptLanguage, currency } = config;
+const createCart = async (context, cartDraft: CartData = {}) => {
```

Comment:
how can I invoke API client method and pass a custom query to it now?

eg
createCart(params, customQuery)